parquet/src/main/java/org/apache/iceberg/parquet/ParquetWriter.java

```diff
@@ -211,6 +228,8 @@ private void flushRowGroup(boolean finished) {
         writer.startBlock(recordCount);
         writeStore.flush();
         pageStore.flushToFileWriter(writer);
+        totalRawBufferedSize += currentRawBufferedSize;
+        totalRowGroupSize += writeStore.getBufferedSize();
```

Comment:
I'm not sure I understand this part, what are we adding to totalRowGroupSize here?